src/components/Filters/Indicators.js

```diff
@@ -31,7 +48,7 @@ class Indicators extends Component {
         this.setState({children})
     }
 
-    handleChange = (e) => {
+    handleChange = (e: any) => {
```

Comment:
Note that `antd` does not have a built flowtype for this. It uses the TypeScript https://github.com/HopkinsIDD/covid-dashboard-app/pull/86/files#diff-ecd2affe445bc1a8b13c4648a0bf3550R4


That being said,  the flow community attempts to add types for popular libraries so it is possible to get types for most popular project. It requires pulling down a flow-typed file https://github.com/flow-typed/flow-typed/blob/master/definitions/npm/antd_v3.x.x/flow_v0.104.x-/antd_v3.x.x.js